.github/workflows/pr_validate.yaml

```diff
@@ -28,7 +28,7 @@ jobs:
       - id: check-issue-url
         name: Check issue URL
         run: |-
-          REGEX="#### URL to issue\s?\r\n(https:\/\/github\.com\/hms-dbmi-cellenics\/issues\/issues\/[0-9]+|N\/A)"
+          REGEX="#### URL to issue\s?\r\n(https:\/\/github\.com\/hms-dbmi-cellenics\/issues\/issues\/[0-9]+|https:\/\/biomage\.atlassian\.net\/browse\/BIOMAGE-[0-9]+|https:\/\/github\.com\/biomage-org\/issues\/issues\/[0-9]+|N\/A)"
```

Comment:
Why is it necessary here to check for issues in either `biomage-org` or Biomage's private Jira (which is not publicly viewable)?